src/resources/postcss/components/_datepicker.pcss

```diff
@@ -0,0 +1,135 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Datepicker
+ *
+ * ----------------------------------------------------------------------------- */
+
+.tribe-events {
+
+	.datepicker {
+		background-color: var(--color-background);
+		border: 1px solid var(--color-border-default);
+		box-shadow: var(--box-shadow-tooltip);
+		margin-top: var(--spacer-1);
+		padding: 0 var(--spacer-3) var(--spacer-3);
+
+		@media (--viewport-medium) {
+			margin-top: var(--spacer-3);
+		}
+
+		&:before,
+		&:after {
+			display: none;
+		}
+
+		td,
+		th {
+			height: initial;
+			width: initial;
+		}
+
+		.dow {
+			color: var(--color-text-primary);
+			font-size: var(--font-size-0);
+			font-weight: var(--font-weight-regular);
+			padding: initial;
+			padding-bottom: var(--spacer-2);
+		}
+
+		.datepicker-switch {
+			color: var(--color-text-primary);
+			font-size: var(--font-size-3);
+			font-weight: var(--font-weight-regular);
+		}
+
+		.datepicker-switch,
+		.prev, .next {
+			line-height: initial;
+			padding: initial;
+			padding: var(--spacer-5) 0;
+
+			&:hover,
+			&:focus,
+			&:active {
+				background-color: var(--color-background);
+			}
+		}
+
+		.prev,
+		.next {
+			text-indent: -999em;
+			position: relative;
+
+			&:before {
+				height: 14px;
+				width: 14px;
+				content: '';
+				background-repeat: no-repeat;
+				background-size: contain;
+				display: block;
+				position: absolute;
+			}
+		}
+
+		.prev {
+			&:before {
+				background-image: svg-inline(caret-left);
+				left: 0;
+			}
+
+			&:hover,
+			&:focus,
+			&:active {
+				&:before {
+					background-image: svg-inline(caret-left-active);
+				}
+			}
+		}
+
+		.next {
+			&:before {
+				background-image: svg-inline(caret-right);
+				right: 0;
+			}
+
+			&:hover,
+			&:focus,
+			&:active {
+				&:before {
+					background-image: svg-inline(caret-right-active);
+				}
+			}
+		}
+
+		.datepicker-switch {
+			font-weight: var(--font-weight-regular);
+		}
+
+		.day {
+			font-size: var(--font-size-3);
+			color: var(--color-text-primary);
+			padding: 14px;
+
+			&.old {
+				color: var(--color-text-disabled);
+			}
+		}
+
+		.month,
+		.year {
+			font-size: var(--font-size-1);
+		}
+
+		.day,
+		.month,
+		.year {
+			&:hover,
+			&:focus,
+			&:active,
+			&.focused {
+				color: #fff;
```

Comment:
we have a variable for this.

Reply:
Yes, for a background, not related. If we change the background on the design system it'll affect this.

Reply:
I've used the variable background color for these cases too. The background color of the datepicker is also the background color, and if that changes to, say, light grey, I would expect the color of the text to be such too in this case.

But if you want to create a new variable for this too I wouldn't be opposed. I do, however, think they are linked.

Reply:
I feel it's confusing, but if you already used it for something similar I don't mind using it here.